<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
      <string>뷰탭순위확인기 v3 by SHW</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>시작</string>
   </property>
  </widget>
  <widget class="QListWidget" name="logList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
        <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
        <y>260</y>
     <width>761</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="keywordRemainIndicator">
   <property name="geometry">
    <rect>
     <x>10</x>
        <y>290</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>현재 키워드 수 / 남은 키워드 수 : 0 / 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentkeywordIndicator">
   <property name="geometry">
    <rect>
     <x>240</x>
        <y>290</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>검색 중인 키워드 : </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>761</width>
     <height>381</height>
    </rect>
   </property>
      <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="autoScroll">
          <bool>true</bool>
      </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
       <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
       <number>253</number>
   </attribute>
   <column>
    <property name="text">
     <string>키워드</string>
    </property>
   </column>
      <column>
          <property name="text">
              <string>url</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>순위</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="allCheckBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>뷰탭 전체 검색</string>
   </property>
   <property name="checked">
       <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
        <x>600</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
    </rect>
   </property>
      <property name="text">
          <string>(체크 해제시, 1 page 검색)</string>
      </property>
  </widget>
     <widget class="QLabel" name="label_2">
         <property name="geometry">
             <rect>
                 <x>600</x>
                 <y>120</y>
                 <width>161</width>
                 <height>141</height>
             </rect>
         </property>
         <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/SNB_logo_nobg.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
     </widget>
     <widget class="QLabel" name="label_3">
         <property name="geometry">
             <rect>
                 <x>602</x>
                 <y>112</y>
                 <width>91</width>
                 <height>16</height>
             </rect>
         </property>
         <property name="text">
             <string>순위 출력 제한 :</string>
         </property>
     </widget>
     <widget class="QLineEdit" name="rankingLimit">
         <property name="geometry">
             <rect>
                 <x>700</x>
                 <y>110</y>
                 <width>41</width>
                 <height>20</height>
             </rect>
         </property>
         <property name="text">
             <string>0</string>
         </property>
     </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
